<?xml version="1.0"?>

<launch>

<arg name="global_frame" default="/map"/>
<arg name="robot_base_frame" default="base_link"/>

    <!-- <node pkg="navfn" type="navfn_node" respawn="false" name="planner" output="screen">
        <param name="navfn_planner/visualize_potential" value="true" />
        <param name="navfn_planner/allow_unknown" value="true" />
        <param name="navfn_planner/planner_window_x" value="0.0" />
        <param name="navfn_planner/planner_window_y" value="0.0" />
        <param name="navfn_planner/default_tolerance" value="0.0" />
        <remap from="/planner/goal" to="/move_base_simple/goal" />
    </node> -->

    

            
    

    <!-- <node pkg="costmap_2d" type="costmap_2d_node" name="costmap_node" output="screen" >
        <param name="costmap/global_frame" value="$(arg global_frame)"/>
        <param name="costmap/robot_base_frame" value="$(arg robot_base_frame)"/>
        <param name="costmap/origin_x" value="-5.0"/>
        <param name="costmap/origin_y" value="-5.0"/>

        <param name="costmap/width" value="10.0"/>
        <param name="costmap/height" value="10.0"/>
        <param name="costmap/resolution" value="0.05"/>
        <param name="costmap/transform_tolerance" value="0.2"/>
        <param name="costmap/update_frequency" value="5.0"/>
        <param name="costmap/publish_frequency" value="0.0"/>
        <param name="costmap/transform_tolerance" value="0.2"/>
        <param name="costmap/rolling_window" value="false"/>
        <param name="costmap/track_unknown_space" value="false"/>
        <param name="costmap/always_send_full_costmap" value="false"/>
        <param name="costmap/footprint_padding" value="0.0"/>
        <param name="costmap/robot_radius" value="0.5"/>
    </node> -->


</launch>
